runtime/src/lib.rs

```diff
@@ -262,6 +271,58 @@ impl frame_system::Config for Runtime {
 	type MaxConsumers = frame_support::traits::ConstU32<16>;
 }
 
+pub struct TagProvider;
+impl pallet_account_alias_registry::TagProvider<Runtime> for TagProvider {
+	fn tag(id: &AccountId, name: &str) -> Result<u16, ()> {
+		let salt = pallet_timestamp::Pallet::<Runtime>::get() / 86400000u64;
```

Comment:
```suggestion
		let salt = pallet_timestamp::Pallet::<Runtime>::get() >> 26u64;
```

Let's use shift here simply. `log2(86,400,000) =~ 26.3645`